File: Practica1/Backend-python/app/repositories/movie_repository.py
from app.config import get_connection

def get_all_movies():
    conn = get_connection()
    cursor = conn.cursor()

    cursor.execute("""
        SELECT id_pelicula, titulo, director, anio_estreno,
               url_contenido, poster, estado
        FROM peliculas
        ORDER BY id_pelicula DESC
    """)

    movies = cursor.fetchall()

    cursor.close()
    conn.close()

    return movies

File: Practica1/Backend-Nodejs/src/repositories/movie.repository.js
const pool = require("../config/db");

async function getAllMovies() {
  const query = `
    SELECT 
      id_pelicula,
      titulo,
      director,
      anio_estreno,
      url_contenido,
      poster,
      estado
    FROM peliculas
    ORDER BY id_pelicula DESC
  `;

  const result = await pool.query(query);
  return result.rows;
}

module.exports = {
  getAllMovies
};